### DISCUSSION TITLE
BLE - Integration - HA Coded Associated Device Trackers Flap - home/away

### DISCUSSION THREAD

**Discussion by bleitchhk** [mKeRix/room-assistant#524] [Q&A]
Hi;

I have been using room-assistant for a while primarily with BLE tags and with a lot of success.  When a BLE tag is away from the property, the associated device tracker in HA reliably keeps it set to "away"

I have a small issue at home however.  Frequently, 1 or more of the BLE tags will "flap" between home and away.

My setup is as follows:

- 5 Room Assistant instances on 5x Pi 0ws
- in 5 different areas
- using BLE and Home Assistant Core integrations only
- using Bluecharm BLE beacons for 4 sets of keychains
- MQTT up and running solid and reliably

- Tag setup:
    - advertisement interval - 100ms
    - Tx Power   (-16)
    - Measure Power (-59)
    - TLM Interval  (10)

(BLE Beacon Screenshot png attached)

![Ble beacon config](https://user-images.githubusercontent.com/71584723/107408813-4d6a2880-6ac0-11eb-99f3-baac7dc5f784.png)

- Room assistant setup (cut and past exact)

cluster:
  quorom: 4
  weight: 500
  autoDiscovery: false
  port: 6425
  peerAddresses:
    - 192.168.nn.nn:6425
    - 192.168.nn.nn:6425
    - 192.168.nn.nn:6425
    - 192.168.nn.nn:6425
    - 192.168.nn.nn:6425

global:
  integrations:
    - homeAssistant
    - bluetoothLowEnergy
#
homeAssistant:
  mqttUrl: 'mqtt://192.168.nn.nn:1883'
  mqttOptions:
    username: xxxxxxxxx
    password: xxxxxxxxxx
#
bluetoothLowEnergy:
  whitelist:
    - 426c7565436861726d426561636f6a73-3838-4949
    - 426c7565436861726d426561636f6b73-3838-4950
    - 426c7565436861726d426561636f6c73-3838-4951
    - 426c7565436861726d426561636f6d73-3838-4952
    - 426c7565436861726d426561636f6e73-3838-4953
    - 426c7565436861726d426561636f6f73-3838-4954

the exact behaviour is that the tag will switch sometimes from "away" to "home" and back a few times in 2-8 second intervals.

Any help / guidance appreciated

**Comment by mKeRix** [comment]
Have you tried increasing the `timeout` for the `bluetoothLowEnergy` integration already? Based on your description I would try to set it to `15` or higher.
